### PR TITLE
fix(reports): fix profit/loss statement

### LINKED ISSUE #6180: Updates for the Profit And Loss Statement report
Finances > Report > Profit And Loss Statement

Not sure if this report is working.  I tried the Vanga dataset and bhima_test with various period ranges and all the numbers are zero.  Not sure if this a bug or some configuration issue.  It should probably be tested with other production datasets and debugged if they show zeros too.

**Suggestions**
- Increase the font size of the date range on the report
- The format of the rest of the report looks fine

### PR DESCRIPTION
**Pull request by jniles**
This commit fixes the profit and loss statement by correctly using the Tree library.

Closes #6180.

### CODE REVIEW

File: server/controllers/finance/reports/income_expense/index.js [hunk 1]
```diff
@@ -35,98 +35,96 @@ const DEFAULT_PARAMS = {
...
+    const { periodFromId, periodToId, fiscalYearId } = options;
+    const data = {};
+
+    const [periodFrom, periodTo, fiscalYear] = await Promise.all([
+      getPeriodById(periodFromId),
+      getPeriodById(periodToId),
+      Fiscal.lookupFiscalYear(fiscalYearId),
+    ]);
+
+    if (periodFrom.start_date > periodTo.start_date) {
```
Comment (jmcameron) [inline server/controllers/finance/reports/income_expense/index.js:52]: I wonder if this check could be incorporated into bh-fiscal-period-select (in another PR)?

File: server/controllers/finance/reports/income_expense/report.handlebars [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{{> head title="REPORT.PROIFT_AND_LOSS" }}
+{{> head title="REPORT.PROFIT_AND_LOSS" }}
```
Comment (jmcameron) [inline server/controllers/finance/reports/income_expense/report.handlebars:1]: 👍

**Review by jmcameron — APPROVED** [review]
Looks great!

File: server/controllers/finance/reports/income_expense/index.js (hunk 1 above)
Comment (jniles) [reply server/controllers/finance/reports/income_expense/index.js:52]: This is a good idea.  I see you made an issue #6233

**Pull request merged**